### ISSUE TITLE
Information about multi class training with loss_mmod on standard dataset

### ISSUE THREAD

**Issue by cricket1** [davisking/dlib#1829]
I was seeking results of training with loss_mmod on a standard dataset to compare  with ssd ,yolo etc.

In the discussion here [https://www.reddit.com/r/computervision/comments/71yssd/fast_multiclass_object_detection_in_dlib_197/ ](url) there is a mention of dlib being trained on kitti .There is also a link  for results and code [http://www.cvlibs.net/datasets/kitti/eval_object_detail.php?&result=dd1e2a5d6e24180773c32214c4941940480585f6](url)

May be the link is old ,I dont find any results or code there.

Any help here is appreciated

Thanks in advance

**Comment by davisking** [comment]
Unfortunately, the KITTI results page has been reset and I didn't keep an archive of it.  

But you aren't talking about comparing things that are comparable anyway.  SSD and YOLO are not losses.  For instance, you could use the MMOD loss to train the SSD model if you wanted to.

**Comment by cricket1** [comment]
Oh I should have framed the question more appropriately , my bad.

Do you still remember how were your results with KITTI  . Also depth(No of convolution layers) for that model.

**Comment by arrufat** [comment]
Hi, I had the same question some time ago, but luckily the source code is still available here , so you can run the experiment yourself. The model is the same as the one used for the find cars examples, which means only 7 convolutional layers.

**Comment by davisking** [comment]
@cricket1 I don't recall what the results were.

**Comment by davisking** [comment]
@arrufat, sweet.  I forgot all about that. Apparently I did record the whole submission, results and all :).  Look where @arrufat linked @cricket1 and you will find the results.  

I will point out that I think, generally speaking, the model trained on kitti data is worse than the one in the example programs and shown in these videos http://blog.dlib.net/2017/08/vehicle-detection-with-dlib-195_27.html.  This is because the training datasets I used in the dlib example programs are more carefully annotated for this task, which always makes a big difference in the resulting model.  But even so, the kitti trained model worked fine.
